charts/linkurious-enterprise/templates/deployment.yaml

```diff
@@ -105,25 +105,29 @@ spec:
           {{- if .Values.offlineMaintenanceModeEnabled }}
           # Deactivate probes in Maintenance mode
           {{- else }}
+          {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-              path: /api/status/
+              path:  {{ tpl .Values.livenessProbe.path . }}
               port: http
             initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
             timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
             successThreshold: {{ .Values.livenessProbe.successThreshold }}
             failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: /api/status
+              path: {{ tpl .Values.readinessProbe.path . }}
```

Comment:
why use tpl ?

Reply:
Because we are doing this for people who will be using basepath. this enables them to set a basepath and then construct the api/status path with it.